qa/L0_metrics/metrics_queue_size_test.py

```diff
@@ -194,6 +221,13 @@ def test_dynamic_batch_scheduler(self):
         batch_size = self.max_batch_size
         self._test_helper(model_name, batch_size, self._send_async_requests)
 
+    def test_fail_max_queue_size(self):
+        model_name = "max_queue_size"
+        batch_size = self.max_batch_size
+        self._test_helper(
+            model_name, batch_size, self._send_async_requests, max_queue_size=4
+        )
```

Comment:
Can you add a comment on test expectations? At a glance it looks like it wants to fill up the queue to max queue size and assert further requests are rejected/failed, but would be nice to have that summary associated with the test block.

Reply:
LGTM otherwise, just want the description for future readers

Reply:
Done!